Examples/KerrBH/params.txt

```diff
@@ -70,5 +99,4 @@ kappa3 = 0
 covariantZ4 = 0 # 0: default. 1: dampk1 -> dampk1/lapse
 
 # coefficient for KO numerical dissipation
-# NB must be less than 0.5 for stability
```

Comment:
Yes :-) Maybe we should add the ref to Alcubierre until we implement the auto check?

Reply:
Ok, done (for all params files).